README.md

```diff
@@ -71,65 +72,96 @@ Provides the following HTTP methods for `/load-test` endpoint:
 
  If you prefer to use Postman you can also import [openapi.json](openapi.json) file into Postman to create a new collection.
 
-### Kangal controller
-The general name for several Kubernetes controllers created to manage all the aspects of the performance testing process.
- - LoadTest controller  
- - Backend jobs controller
- 
-## To start using Kangal
-To run Kangal in your Kubernetes cluster follow [docs](docs/README.md#how-do-i-use-kangal)
-
-Also check out our [User Flow](docs/Kangal-user-flow.md) guide to start creating load tests with Kangal.
-
-More detailed information can be found in [docs folder](docs/)
-
-## To start developing Kangal
```

Comment:
Why delete this one? It is useful when you want to develop Kangal locally.

Reply:
Not on the first page, but I definitely should not had removed, I will move somewhere inside `docs/`

Reply:
done! https://github.com/hellofresh/kangal/pull/31/commits/1c3dca576eeaf0757bcf817ed38846e055791e4f